### ISSUE TITLE
feature request: Organize import using Biome.js for typescript, javascript, typescriptreact, javascriptreact

### ISSUE THREAD

**Issue by rahulpeacock** [stevearc/conform.nvim#585]
### Did you check existing requests?

- [X] I have searched the existing issues

### Describe the feature

I want to organize the imports using `biome.js` when I format my buffer/file using `biome.js` for typescript, javascript, typescriptreact, javascriptreact

### What is the significance of this feature?

strongly desired

### Additional details

For now I am using the `args` to organize the imports. But it is highly recommend to have a config to organize the imports using biome

This is a workaround for now
```lua
-- confirm.lua
return {
  {
    "stevearc/conform.nvim",
    opts = {
      formatters_by_ft = {
        lua = { "stylua" },
        ["javascript"] = { "biome" },
        ["javascriptreact"] = { "biome" },
        ["typescript"] = { "biome" },
        ["typescriptreact"] = { "biome" },
        ["json"] = { "biome" },
        ["css"] = { "biome" },
      },
      formatters = {
        biome = {
          command = "biome",
          args = { "check", "--formatter-enabled=true", "--linter-enabled=false", "--organize-imports-enabled=true", "--write", "--stdin-file-path", "$FILENAME" },
        },
      },
    },
  },
}
```

**Comment by stevearc** [comment]
According [to the docs](https://biomejs.dev/guides/configure-biome/) it looks like the formatter, linter, and import sorter are all enabled by default. The default conform `biome` formatter calls `biome format`. If you want to use `biome check`, use the `biome-check` formatter.

Also, to make changes to a formatter args, you can provide `append_args` or `prepend_args` instead of overriding the args entirely.

**Comment by rahulpeacock** [comment]
@stevearc Yes I can pass args but I can't pass the arg `--import-sort-enable` to the `biome format` command because it doesn't support that arg

**Comment by rahulpeacock** [comment]
@stevearc can you share any config with `biome-check`.

**Comment by andrew-t-james-core** [comment]
@rahulpeacock I had some luck with this [setup](https://github.com/LazyVim/LazyVim/issues/2116#issuecomment-2489562068).

**Comment by rahulpeacock** [comment]
@andrew-t-james-core this is also a workaround but is there a default config from `conform` ??

**Comment by stevearc** [comment]
Can you not just use the built-in `biome-check` formatter? It runs `biome check` which, from my understanding, will organize the imports
```lua
      formatters_by_ft = {
        lua = { "stylua" },
        ["javascript"] = { "biome-check" },
        ["javascriptreact"] = { "biome-check" },
        ["typescript"] = { "biome-check" },
        ["typescriptreact"] = { "biome-check" },
        ["json"] = { "biome-check" },
        ["css"] = { "biome-check" },
      },
```

**Comment by work-in-progress-danny** [comment]
> Can you not just use the built-in `biome-check` formatter? It runs `biome check` which, from my understanding, will organize the imports
> 
>       formatters_by_ft = {
>         lua = { "stylua" },
>         ["javascript"] = { "biome-check" },
>         ["javascriptreact"] = { "biome-check" },
>         ["typescript"] = { "biome-check" },
>         ["typescriptreact"] = { "biome-check" },
>         ["json"] = { "biome-check" },
>         ["css"] = { "biome-check" },
>       },

This works for me 🚀

**Comment by KMohZaid** [comment]
> Can you not just use the built-in `biome-check` formatter? It runs `biome check` which, from my understanding, will organize the imports
> 
>       formatters_by_ft = {
>         lua = { "stylua" },
>         ["javascript"] = { "biome-check" },
>         ["javascriptreact"] = { "biome-check" },
>         ["typescript"] = { "biome-check" },
>         ["typescriptreact"] = { "biome-check" },
>         ["json"] = { "biome-check" },
>         ["css"] = { "biome-check" },
>       },

idk why `biome-organize-imports` not working but `biome-check` does work
